scipy/signal/signaltools.py

```diff
@@ -4395,6 +4395,43 @@ def decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=True):
     The ``zero_phase`` keyword was added in 0.18.0.
     The possibility to use instances of ``dlti`` as ``ftype`` was added in
     0.18.0.
+    
+    Examples
+    --------
+    
+    >>> from scipy import signal
+    
+    Define wave parameters.
+    
+    >>> wave_duration = 3
+    >>> sample_rate = 100
+    >>> freq = 2
+    >>> q = 5
+    
+    Calculate number of samples.
+    
+    >>> samples = wave_duration*sample_rate
+    >>> samples_decimated = int(samples/q)
+    
+    Create cosine wave.
+    
+    >>> x = np.linspace(0, wave_duration, samples, endpoint=False)
+    >>> y = np.cos(x*np.pi*freq*2)
+    
+    Decimate cosine wave.
+    
+    >>> ydem = signal.decimate(y, q)
+    >>> xnew = np.linspace(0, wave_duration, samples_decimated, endpoint=False)
+    
+    Plot original and decimated waves.
+    
+    >>> import matplotlib.pyplot as plt
```

Comment:
small style issue here: can you put this import at the top of the example, together with the other import?

Reply:
Yes, definitely.  I have removed whitespaces and grouped the imports together.